man/coredump.conf.xml

```diff
@@ -112,13 +112,25 @@
       <varlistentry>
         <term><varname>EnterNamespace=</varname></term>
 
-        <listitem><para>Controls whether <command>systemd-coredump</command> will attempt to use the mount tree of
-        a process that crashed in PID namespace. Access to the namespace's mount tree might be necessary to generate
-        a fully symbolized backtrace. If set to <literal>yes</literal>, then <command>systemd-coredump</command> will
-        obtain the mount tree from corresponding mount namespace and will try to generate the stack trace using the
-        binary and libraries from the mount namespace. Note that the coredump of the namespaced process might
-        still be saved in <filename>/var/lib/systemd/coredump/</filename> even if <varname>EnterNamespace=</varname>
-        is set to <literal>no</literal>. Defaults to <literal>no</literal>.</para>
+        <listitem><para>For processes belonging to a PID namespace, controls whether
+        <command>systemd-coredump</command> shall attempt to process core dumps on the host, using debug
+        information from the namespace of the process that crashed. Access to the
+        namespace's mount tree might be necessary to generate a fully symbolized backtrace. If set to
+        <literal>yes</literal>, then <command>systemd-coredump</command> will obtain the mount tree from
+        corresponding mount namespace and will try to generate the stack trace using the debug information of
+        binaries and libraries contained in that hierarchy. Note that the coredump of the namespaced
+        process might still be saved in <filename>/var/lib/systemd/coredump/</filename> on the host even if
+        <varname>EnterNamespace=</varname> is set to <literal>no</literal>. Defaults to
+        <literal>no</literal>, to maximize security.</para>
```

Comment:
This does not say what "no" means. Does it not process the coredump or processes it the host namespace? Also, it'd be nice to explicitly cover the effect of this setting for a process which didn't use a PID namespace.